### PR TITLE
LINDA Fixes

### PR DESCRIPTION
**Pull request by Fox-McCloud**
Just some LINDA fixups.

When LINDA was ported, a few things were missed, in addition to a few fixes that came after TG's subsystem rewrite (but never the less had nothing to do with the subsystem rewrite).

This pretty much brings LINDA up to TG's standard prior to StonedOne's silly changes.

This should help behaviors of gases in some instances and also increase overall performance of LINDA.

Another thing it allows is for fires on the shuttle (YAY!)...because who doesn't want to set everyone on fire at round end?

@Tigercat2000 for review

### CODE REVIEW

File: code/LINDA/LINDA_turf_tile.dm [hunk 1]
```diff
@@ -352,8 +352,9 @@ atom/movable/proc/experience_pressure_difference(pressure_difference, direction)
 	for(var/turf/simulated/T in turf_list)
 		T.excited = 0
 		T.recently_active = 0
+		T.excited_group = null
```
Comment (tigercat2000) [inline code/LINDA/LINDA_turf_tile.dm]: If this calls garbage_collect() afterwards, why is this necessary? garbage_collect sets the turf excited group to null as well.

File: code/game/turfs/simulated/floor_types.dm [hunk 2]
```diff
@@ -84,23 +92,22 @@
...
-		var/datum/gas/sleeping_agent/trace_gas = new
+/turf/simulated/floor/engine/n20/New()
+	..()
+	var/datum/gas_mixture/adding = new
+	var/datum/gas/sleeping_agent/trace_gas = new
 
-		trace_gas.moles = 2000
-		adding.trace_gases += trace_gas
-		adding.temperature = T20C
+	trace_gas.moles = 6000
```
Comment (tigercat2000) [inline code/game/turfs/simulated/floor_types.dm]: I think this was intentionally 2000, for a limited supply of n20, since it can be dangerous- *with ZAS.* But with LINDA, it's probably fine.

**Comment by tigercat2000** [comment]
Other than like, the one issue I mentioned on a comment, everything else seems to be an oversight. I believe I left the spawn_T20C stuff out because nothing used it.

Going to give this an overall :+1:

File: code/LINDA/LINDA_turf_tile.dm (hunk 1 above)
Comment (Fox-McCloud) [inline code/LINDA/LINDA_turf_tile.dm]: I or you can talk to Aran about this, but he definitely set it to be this intentionally: https://github.com/tgstation/-tg-station/commit/ea5ec6aec2d54e662f69bf3b332c98b5a89102f2#diff-66ee58580795f0aa9d72744f56e1bd6dR357

**Comment by Fox-McCloud** [comment]
Yeah, it's not used currently, but I figured we might as well make it feature complete as a "just in case".
